src/Pages/LikedVideos/LikedVideos.jsx

```diff
@@ -0,0 +1,99 @@
+import React,{ useEffect } from 'react'
+import { Link } from "react-router-dom"
+import axios from "axios"
+import jwt_decode from "jwt-decode"
+import { useLocation } from "react-router-dom"
+import './LikedVideos.css'
+import {
+  Sidebar,
+  useLikedVideos,
+  VideoCard
+} from '../../index'
+import Lottie from "react-lottie"
+import SadboxLottie from "../../Assets/lottie/sad-empty-box.json"
+
+function LikedVideos() {
+
+  const { likedVideosList, dispatchLikedVideosList } = useLikedVideos()
+  const { pathname } = useLocation();
+
+  let sadboxObj = {
+      loop: true,
+      autoplay: true,
+      animationData : SadboxLottie,
+      rendererSettings: {
+        preserveAspectRatio: 'xMidYMid slice'
+      }
+  }
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [pathname]);
+
+  useEffect(()=>{
+
+    const token=localStorage.getItem('token')
+
+    if(token)
+    {
+      const user = jwt_decode(token)
+      if(!user)
+      {
+        localStorage.removeItem('token')
+      }
+      else
+      {
+        (async () => {
+          let updatedUserInfo = await axios.get(
+            "https://videoztron.herokuapp.com/api/user",
+            {
+              headers:
+              {
+                  'x-access-token': localStorage.getItem('token'),
+              }
+            }
+          )
```

Comment:
I will definitely do these extractions when I start polishing the project after the deadline. 
Because I am well aware there are lot of them.
But thanks for pointing out, will extract them for the new code I write. 😅